cli/medperf/entities/cube.py

```diff
@@ -220,13 +221,16 @@ def download_image(self):
             # Retrieve image from image registry
             logging.debug(f"Retrieving {self.id} image")
             cmd = f"mlcube configure --mlcube={self.cube_path}"
-            with pexpect.spawn(cmd, timeout=config.mlcube_configure_timeout) as proc:
-                proc_out = proc.read()
+            cmd_env = {**os.environ, 'DOCKER_CLI_HINTS': 'false'}
+            with pexpect.spawn(cmd, timeout=config.mlcube_configure_timeout, env=cmd_env) as proc:
+                proc.logfile_read = DebugAndOutputLogger()  # Redirect cmd output to medperf stdout
```

Comment:
I was exploring to use `combine_proc_sp_text` there to display docker logs in a handy way. However, this function does not work with non-linear output (i.e. when output is modifying in the process with escape sequences), and I wasn't successful to fix that. Thus, offering the current, a few ugly, but still solution